Makefile

```diff
@@ -12,14 +12,14 @@ ${PROG}:	${PROG}.c
 	@OSTYPE=`uname`; \
 	OS_CFLAGS=-D__OS_$${OSTYPE}__; \
 	if [ $$OSTYPE = "FreeBSD" ]; then \
-		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE ${HAS_FREEBSD_ACL}"; \
+		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE -DHAS_UTIMENSAT ${HAS_FREEBSD_ACL}"; \
```

Comment:
According to the manpage, FreeBSD only got `utimensat` support in release 10.3.  So we should probably define `HAS_UTIMENSAT` support in pjdfstest.c along with the other version-dependent `HAS_*` variables